### DISCUSSION TITLE
Missing admin.allowEdit in upload field

### DISCUSSION THREAD

**Discussion by GermanJablo** [payloadcms/payload#9564] [Feature Requests & Ideas]
In this PR allowEdit was introduced for the relationship field: https://github.com/payloadcms/payload/pull/8398

The same needs to be done for the upload field.

**Comment by GermanJablo** [comment]
This Issue was created for consistency since the upload and relationship fields have similarities. 
However, no one ever actually requested it. 
I will move it to a discussion as a feature request and implement it if there is even some demand.

**Comment by ivkrpv** [comment]
hm.. was trying to find a way to hide that "edit" button on our upload fields and it seems we actually need this feature :) would be great to have it, then!

ps: is there any way currently to hide it? tried with the "access.update" function, but it hides both "edit" and "delete" buttons.
